### PR TITLE
[LIVY-635][BUILD] Fix travis fail to build by excluding the dependency of hbase

### PR DESCRIPTION
**Pull request by runzhiwang**
## What changes were proposed in this pull request?

Fix travis fail to build by excluding the dependency of hbase.

## How was this patch tested?

existed UT and IT.

### CODE REVIEW

File: thriftserver/server/pom.xml [hunk 1]
```diff
@@ -39,6 +39,10 @@
           <groupId>tomcat</groupId>
           <artifactId>*</artifactId>
         </exclusion>
+        <exclusion>
+          <groupId>org.apache.hbase</groupId>
```
Comment (yiheng) [inline thriftserver/server/pom.xml]: This dependency scope is not test. Not sure if this exclusion is safe

**Comment by jerryshao** [comment]
CC @mgaido91 , would you please help to review. Thanks!

**Comment by jerryshao** [comment]
@runzhiwang can you please also verify if thriftserver can be worked  without issue?

**Comment by runzhiwang** [comment]
@jerryshao It's ok to work.

**Comment by jerryshao** [comment]
I see, thank!

**Comment by mgaido91** [comment]
LGTM too, this should be safe as AFAIK, dependencies on HBase are needed only for using HBase as a metastore, but we do not use any metastore at all. Thanks.

**Pull request closed without merging**